src/en/about/features.md

```diff
@@ -3,6 +3,12 @@
 - Open source, [AGPL License](/LICENSE).
 - Self hostable, easy to deploy.
   - Comes with [Docker](#docker) and [Ansible](#ansible).
+- Federated with Mastodon and Pleroma.
+  - View Lemmy communities, user profiles, posts and comments.
+  - Follow Lemmy communities to receive new posts and comments.
+  - Replies (mentions) work in both directions, including notifications.
+  - Pleroma users can exchange private messages with Lemmy users.
+  - Projects whose federation works similar to Pleroma/Mastodon will likely also federate.
```

Comment:
More importantly, it federates with other Lemmy instances. In that case practically everything works via federation, even moderating a community on another instance.

Reply:
That could maybe be, "Federates with itself, and other fediverse projects like mastodon and pleroma"

Reply:
Its still worth listing which functionality works with other projects. But i guess that fits better in a different part of the docs.

Reply:
Ya, as this is just listing the main features, I'd def prefer just one line here mentioning federation, and put more detailed things elsewhere.